tol_lab_share/messages/mlwh/create_aliquot_message.py

```diff
@@ -103,15 +103,15 @@ def to_string(self) -> str:
 
         return json.dumps({"lims": self.lims, "aliquot": self.aliquot.to_dict()})
 
-    def publish(self, publisher: BasicPublisher, exchange: str, lims_uuid: str) -> None:
+    def publish(self, publisher: BasicPublisher, exchange: str, aliquot_uuid: str) -> None:
         """Publish a new message in the queue with the current contents of the feedback message
         Parameters:
         publisher (BasicPublisher) instance of basic publisher that we will use to connect to rabbitmq
         schema_registry (SchemaRegistry) instance of schema registry that we will use to retrieve data from redpanda
         exchange (str) name of the exchange where we will publish the message in Rabbitmq
         """
         message = self.to_string()
-        routing_key = self._prepare_routing_key(lims_uuid)
+        routing_key = self._prepare_routing_key(aliquot_uuid)
```

Comment:
Why does the routing key include the aliquot UUID?

Reply:
The exchanges bound to warehouse queues are topic exchanges with identifiers. I thought of following a similar pattern.

![image](https://github.com/user-attachments/assets/3deaa250-c4f0-4537-a609-d2c8b33a5d1a)

Reply:
Sure thing.  I'm just not sure the UUID belongs in the routing key.  The `#` above can represent zero or more additional words, so you can just include nothing there.  The UUID would never be used to choose how the message is routed from an exchange to a queue.